js/map_utils.js

```diff
@@ -212,8 +212,14 @@ function applyZoomAndStyle(newView) {
     .style("stroke-width",  1/zoom.s); // make all counties have scaled stroke-width
   
   // apply the transform (i.e., actually zoom in or out)
+  var zoomTime;
+  if(waterUseViz.interactionMode === 'hover'){
+    zoomTime = 750;
+  } else {
+    zoomTime = 0;
+  }
```

Comment:
cool, so we'll just have these 3 if-else chunks to edit if we decide to go back to animated transitions on mobile :+1: